kDriveCore/Data/Models/Upload/UploadSession.swift

```diff
@@ -45,7 +45,8 @@ public final class UploadSession: EmbeddedObject, Decodable {
         fileName = try container.decode(String.self, forKey: .fileName)
         result = try container.decode(Bool.self, forKey: .result)
         token = try container.decode(String.self, forKey: .token)
-        uploadHost = try container.decode(String.self, forKey: .uploadHost)
+        let hostWithScheme = try container.decode(String.self, forKey: .uploadHost)
+        uploadHost = hostWithScheme.replacingOccurrences(of: "https://", with: "")
```

Comment:
making sure we just store the host, without the scheme, else it would fail further down the line.